c/meterpreter/source/extensions/stdapi/server/net/config/route.c

```diff
@@ -229,26 +233,49 @@ DWORD add_remove_route(Packet *packet, BOOLEAN add)
 	route.dwForwardDest    = inet_addr(subnet);
 	route.dwForwardMask    = inet_addr(netmask);
 	route.dwForwardNextHop = inet_addr(gateway);
-	route.dwForwardType    = 4; // Assume next hop.
-	route.dwForwardProto   = 3;
+	route.dwForwardType    = MIB_IPROUTE_TYPE_INDIRECT; // Assume next hop.
+	route.dwForwardProto   = MIB_IPPROTO_NETMGMT;
 	route.dwForwardAge     = -1;
+	route.dwForwardMetric1 = 0;

```

Comment:
Should this not be `-1` if this is not being used? According to https://learn.microsoft.com/en-us/windows/win32/api/ipmib/ns-ipmib-mib_ipforwardrow that is the case?

Reply:
Yep np happy to keep in as is 👍